CHANGELOG.md

```diff
@@ -2,13 +2,11 @@
 
 ## [0.0.7] - 2024-06-10
 
-### Added
-
-- Added Base Mainnet network support
-
 ### Changed
+Updated the usage of `Coinbase.networkList` to `Coinbase.networks`
 
-- Updated the usage of `Coinbase.networkList` to `Coinbase.networks`
+### Added
+- Added Base Mainnet network support
 
```

Comment:
Lets format the CHANGELOG.md before merge

Reply:
We also should add the transaction class addition to change log

Reply:
ah, just merged but I'm going to format in my PR

Reply:
no worries. Lets just change before the release.